### DISCUSSION TITLE
Aggregation Pipeline and Operators

### DISCUSSION THREAD

**Discussion by SemAiSoft** [maxnowack/signaldb#1149] [Ideas]
EDIT: Never mind, sorry - didn't see it before. this feature request was already tendered, and the proposed solutions is good enough. THANKS MANN.

I always dream of a  perfect client storage solution with mongo-db like api and server sync mechanism, and alas! 
THIS IS IT, after 7years of waiting. KUDOS to the author. You are great.

Alright, knowing that this library uses `mingo` under the hood, I am surprise not to find it implementing Aggregation pipeline or its operators as offered by the `mingo` - which is much need.

Is this an oversight from me? Or,
design constraints and limitations of this library?
Is there plan for it in the future?

It will be GREAT if custom operators can be registered or plug-in and out, as offered by the `mingo`!
Thanks.

**Comment by maxnowack** [comment]
Just for completion here the link to the related discussion: #924
